sklearn/linear_model/tests/test_logistic.py

```diff
@@ -278,7 +278,7 @@ def test_consistency_path():
 
 def test_liblinear_dual_random_state():
     # random_state is relevant for liblinear solver only if dual=True
-    X, y = make_classification(n_samples=20)
+    X, y = make_classification(n_samples=20, random_state=0)
```

Comment:
As @chenhe95 is new to the project I think it's ok to be allow it  ;) Now he know better.

Thanks for the detective work and fix!